src/github.ts

```diff
@@ -82,7 +82,7 @@ export class GitHubRepository {
     if (params) {
       url.query = params;
     }
-    const resp = await request({
+    const resp = await request<ResponseData>({
```

Comment:
What do think about:

```ts
const resp: ResponseData = await request({
```

I'm debating on whether that is better (or even if TS is smart enough to infer the types on the RHS).  The  benefits I see is that makes it clear to the reader what the type of `resp` is without knowing that the template parameter for `request` is the type it returns.

WDYT?

Reply:
In this case `ResponseData` refers to the type of `resp.data`, so the type annotation as written wouldn't exactly work.  I personally like the generic type we have 🤷‍♂